packages/tocco-ui/src/Popover/Popover.js

```diff
@@ -65,22 +92,38 @@ class Popover extends React.Component {
 }
 
 Popover.defaultProps = {
-  placement: placements.TOP
+  isPlainHtml: true,
+  isRimless: false,
+  placement: placements.TOP,
+  spacer: 10
 }
 
 Popover.propTypes = {
   /**
-   * Content of the popover
+   * Content in the popover.
    */
   content: PropTypes.node,
   /**
-   * Content of the popover
+   * Reference to the popover.
    */
   children: PropTypes.node,
+  /**
+   * Remove space between content and border if useful (e.g. content is an image only). Default is {false}.
+   */
+  isRimless: PropTypes.boolean,
```

Comment:
would it be more consistent if you name this just "rimless"

dense of <Button> also is not called isDense